cmake/opentelemetry-cpp-config.cmake.in

```diff
@@ -32,9 +32,11 @@
 #   opentelemetry-cpp::otlp_recordable                - Imported target of opentelemetry-cpp::otlp_recordable
 #   opentelemetry-cpp::otlp_grpc_exporter             - Imported target of opentelemetry-cpp::otlp_grpc_exporter
 #   opentelemetry-cpp::otlp_grpc_log_exporter         - Imported target of opentelemetry-cpp::otlp_grpc_log_exporter
+#   opentelemetry-cpp::otlp_grpc_metrics_exporter     - Imported target of opentelemetry-cpp::otlp_grpc_metrics_exporter
```

Comment:
Name it as otlp_grpc_metric_exporter?

Reply:
otlp_grpc_metric_exporter is also missed before, I add both otlp_grpc_metric_exporter and otlp_http_metric_exporter here.